packages/services/src/__tests__/helpers.test.ts

```diff
@@ -43,3 +43,147 @@ describe('first', () => {
     expect(result.quantity).toBe(2);
   });
 });
+
+describe('concatService', () => {
```

Comment:
Hi, can we add the test case for an invalid PATH and no arguments are provided?

Reply:
good catch!